workspaces/config/lib/index.js

```diff
@@ -232,6 +232,8 @@ class Config {
     for (const { data } of this.data.values()) {
       this.#flatten(data, this.#flatOptions)
     }
+    this.#flatOptions.nodeBin = process.env.NODE || this.execPath
```

Comment:
Oh yes very much thank you.  When reading this code for this PR it stuck out that we were doing this in the flatten command.